docs/user-guide/tftp.rst

```diff
@@ -0,0 +1,124 @@
+******************
+The TFTP Directory
+******************
+
+For booting machines in a PXE and/or HTTP-Boot environment the TFTP directory is the most important directory. In this
+folder all static files required for booting are living.
+
+The folder of this is dependant on your distro and can be changed in the Cobbler settings. The default should be
+correctly set during the package build of your Linux distro or during the installation process (if you are use the
+source installation).
+
+Behaviour
```

Comment:
I don't have the opinion that I am describing a workflow or procedure here. The code does not directly follow what I have written here. Thus in my eyes this is what the user should see if he watches the folder for changes. Thus this is the externally visible behaviour of the application in my eyes. Please try to explain further if you believe my argumentation is flawed.

Reply:
Completely fine. Maybe I went a bit overboard and interpreted a bit too into that word. "Behaviour" reminds me a more of some human term rather than some machine term. Although it can be also the latter as I learned. Just ignore my question. :slightly_smiling_face:

Reply:
Okay :)